sal_interfaces/OCPS/OCPS_Events.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://project.lsst.org/ts/sal_objects/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://project.lsst.org/ts/sal_objects/schema/SALEventSet.xsd">
+  <SALEvent>
+    <Subsystem>OCPS</Subsystem>
+    <Version>1.0.0</Version>
+    <Author>Kian-Tat Lim (ktl@slac.stanford.edu)</Author>
+    <EFDB_Topic>OCPS_logevent_job_result</EFDB_Topic>
+    <Alias>job_result</Alias>
+    <Explanation>https://dmtn-133.lsst.io/</Explanation>
```

Comment:
This is also going to go away at some point, so you can choose to do that now or wait until later.